app/services/last_updated_snps_service.rb

```diff
@@ -0,0 +1,9 @@
+class LastUpdatedSnpsService
```

Comment:
You can fix all of these running `bundle exec rubocop --auto-correct --only Style/FrozenStringLiteralComment`. You may have to merge master into your branch first.